### DISCUSSION TITLE
Cant deploy my webite

### DISCUSSION THREAD

**Discussion by jennyberry** [thuliteio/doks#865] [Q&A] [answered]
I followed the [docs](https://getdoks.org/docs/how-to/hosting/deployment/#deploy-to-github-pages) 
However, after adding deploy-github.yml in .github/workflows then push to my [repo](https://github.com/jennyberry/easterntownhall) but there are no gh-pages created. My [website](https://jennyberry.github.io/easterntownhall/) is simplely built from README.md. 

I am newbie and still try to get familiar with this env, so please help 🙏🙏🙏
Really appreciate your help!!!
Thank you.

**Comment by h-enk** [comment, marked as answer]
Looks like you got it working now? https://github.com/jennyberry/easterntownhall/actions/runs/2829474876

Note, [GitHub Pages now uses Actions by default](https://github.blog/2022-08-10-github-pages-now-uses-actions-by-default/)

**Reply by jennyberry** [reply]
Thank you for always helping me out. Yes, it's working now. my project failed due to multiple errors but I cloned the new one and deploy it right after that, so I could figure out where I went wrong.

**Comment by h-enk** [comment]
Make sure that `npm test` succeed — see e.g. https://github.com/jennyberry/c3eth-new/runs/7737710339?check_suite_focus=true

**Comment by mysubcult** [comment]
so, how to solve that problem?

**Comment by h-enk** [comment]
See https://github.com/h-enk/doks/discussions/986#discussioncomment-4717601
